adhocracy4/maps/static/a4maps/map_choose_point.js

```diff
@@ -0,0 +1,101 @@
+function createMap (L, baseurl, name) {
+  var basemap = baseurl + '{z}/{x}/{y}.png'
+  var osmAttrib = '&copy; <a href="http://openstreetmap.org/copyright">OpenStreetMap</a> contributors'
```

Comment:
As far as I remember, the attribution changes with `baseurl` (e.g.  [mapbox](https://www.mapbox.com/help/attribution/#text-attribution) requires a specific attribution).